app/views/candidate_interface/degrees/review/show.html.erb

```diff
@@ -12,14 +12,31 @@
       <%= render CandidateInterface::EditableSectionWarning.new(section_policy: @section_policy, current_application: @application_form) %>
 
       <% degree_empty_component = CandidateInterface::DegreeEmptyComponent.new(application_form: @application_form) %>
+
       <% unless degree_empty_component.render? || !@section_policy.can_edit? %>
         <%= govuk_button_link_to t('application_form.degree.another.button'), candidate_interface_degree_country_path(context: :new_degree), secondary: true %>
       <% end %>
     </div>
   </div>
 
   <%= render degree_empty_component %>
-  <%= render CandidateInterface::DegreeReviewComponent.new(application_form: @application_form, editable: @section_policy.can_edit?, deletable: @section_policy.can_delete?) %>
+
+  <% if @application_form.no_degree_and_degree_completed? %>
+    <%= render(SummaryCardComponent.new(
+      rows: [
+        {
+          key: 'Do you have a university degree?',
+          value: 'No, I do not have a degree',
+          action: { href: candidate_interface_degree_university_degree_path },
```

Comment:
Found a bug here. This needs to use the `section_policy.can_edit?` because of the additive changes feature. 
I will transform this small line into a component tomorrow morning.

Reply:
Fixed on 1a429a985e7115da48521a926b3aeb520fe6c309